packages/babel-helpers/src/index.js

```diff
@@ -61,16 +79,18 @@ function getHelperMetadata(file) {
       const bindings = path.scope.getAllBindings();
 
       Object.keys(bindings).forEach(name => {
-        if (name === exportName) return;
+        if (name === exportName || name in dependencies) return;
 
         localBindingNames.add(name);
       });
     },
     ReferencedIdentifier(child) {
       const name = child.node.name;
-      const binding = child.scope.getBinding(name);
-
-      if (!binding) globals.add(name);
+      if (name in dependencies) {
+        importBindingsReferences.push(makePath(child));
```

Comment:
This may not reference the same identifier, need to check binding.

Reply:
Done